packages/rxjs/README.md

```diff
@@ -0,0 +1,84 @@
+# <img src="apps/rxjs.dev/src/assets/images/logos/Rx_Logo_S.png" alt="RxJS Logo" width="86" height="86"> RxJS: Reactive Extensions For JavaScript
+
+![CI](https://github.com/reactivex/rxjs/workflows/CI/badge.svg)
+[![npm version](https://badge.fury.io/js/rxjs.svg)](http://badge.fury.io/js/rxjs)
+[![Join the chat at https://gitter.im/Reactive-Extensions/RxJS](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/Reactive-Extensions/RxJS?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+
+# RxJS 8
+
+### FOR 7.X PLEASE GO TO [THE 7.x BRANCH](https://github.com/ReactiveX/rxjs/tree/7.x)
```

Comment:
If we're gonna have this file here, I don't think we have to mention V7 here.

Nor any mention of the RxJS versions.

Reply:
That can be done in a follow up.